src/Assessors/CyclomaticComplexity/CyclomaticComplexityAssessor.php

```diff
@@ -41,7 +41,7 @@ public function assess(string $filePath): int
      * @param  string $contents File contents.
      * @return void
      */
-    protected function countTheMethods(string $contents)
+    protected function countTheMethods(string $contents): void
```

Comment:
All the `void` return types will have to be removed unfortunately. They are only PHP 7.1+ and I want to keep supporting PHP 7.0.